client/src/css/overview.css

```diff
@@ -35,20 +35,38 @@
   max-width: 100px;
   display: flex;
   flex-flow: column nowrap;
-  align-items: flex-start;
+  align-items: center;
+}
+
+.gallery-thumbnails-viewport {
+  overflow: hidden;
+}
+
+.gallery-thumbnails-all {
+  transform: translate(0, 0);
+  margin: 10px;
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  row-gap: 20px;
+  transition: all 400ms ease;
 }
 
 .main-image-pane {
```

Comment:
You can consider creating a fixed container so that even if the image resolution has changed, it will not adjust your pane. 

e.g. 
```
.main-image-pane {
  /* background-color: #dddddd; */
  width: 692.47px;
  height: 692.47px;
  position: relative;
}
```